perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py

```diff
@@ -45,11 +45,21 @@
 from perfkitbenchmarker import vm_util
 from perfkitbenchmarker.sample import PercentileCalculator  # noqa
 
+
 flags.DEFINE_enum('storage', benchmark_spec_class.GCP,
                   [benchmark_spec_class.GCP, benchmark_spec_class.AWS,
                    benchmark_spec_class.AZURE],
                   'storage provider (GCP/AZURE/AWS) to use.')
 
+flags.DEFINE_string('object_storage_gcp_region', 'us-central1',
```

Comment:
The fact there are three very similar flags here seems to indicate a level of abstraction is missing.

I think we should treat object stores like other resources and have them in the GCP, AWS, .. .directory.  Then these flags could be there.